paddlenlp/transformers/model_utils.py

```diff
@@ -494,42 +495,52 @@ def save_pretrained(self, save_dir):
             logger.warning(
                 "Save pretrained model only supported dygraph mode for now!")
 
-    def resize_token_embeddings(self, new_num_tokens=None):
+    def resize_token_embeddings(self, new_num_tokens: Optional[int]=None) -> nn.Embedding:
         """
         Resizes input token embeddings matrix of the model according to new_num_tokens.
 
         Args:
-            new_num_tokens (int): The number of new tokens in the embedding matrix. Increasing the size will add newly initialized
+            new_num_tokens (Optional[int]):
+                The number of new tokens in the embedding matrix. Increasing the size will add newly initialized
                 vectors at the end. Reducing the size will remove vectors from the end. If not provided or None, just
                 returns a pointer to the input tokens embedding module of the model without doing anything.
 
         Returns:
             paddle.nn.Embedding: The input tokens Embeddings Module of the model.
         """
-        old_embeddings = self.get_input_embeddings()
+        # 1. resize the embedding
+        try:
+            old_embeddings = self.get_input_embeddings()
+        except NotImplementedError:
+            raise NotImplementedError(
```

Comment:
It is not a good choice to catch a `NotImplementedError` and then throw a `NotImplementedError`, maybe what we need to do is just to add error message into `get_input_embeddings`

Reply:
Yes,  we should add error message into the `get_input_embedding` method.